docs/Proxy.md

```diff
@@ -0,0 +1,25 @@
+# Proxy
+
+This page explains how to get vscode-xml to work through a proxy.
+
+## Prerequisites
+
+If you want to use the binary language server,
+then you will need to make sure that your proxy allows connections to jboss.org
+since the binary language server is downloaded from that website.
+You will also need to make sure that you can access any online XML schemas that you want to work with through the proxy.
+
+## Setting up the proxy in VS Code
+
+VS Code provides a setting called `http.proxySupport`.
+If you set it to `override`, VS Code will attempt to rewrite the requests from extensions to go through the proxy that is configured for VS Code.
+However, this method sometimes does not work correctly.
+
+The tested way to get vscode-xml to use your proxy is to set the following settings:
+ * `http.proxy`
+ * `http.proxyAuthorization`, if you use an authenticated proxy
+
+Once these settings are set,
```

Comment:
Does LemMinX respect the VS Code proxy settings ? I see the JVM has a separate set of properties.

Reply:
vscode-xml will check if the user configured JVM args has the parameters to set up the proxy. If they don't, vscode-xml converts the values of these settings into the necessary JVM args and appends them to the existing args.

Reply:
Right. I can see that in settings/proxySettings.ts.